internal/engine/configs/api.go

```diff
@@ -205,6 +207,29 @@ func toCmdObjects(tlr tiltfile.TiltfileLoadResult) typedObjectSet {
 	return result
 }
 
+// Pulls out all the UIResource objects generated by the Tiltfile.
+func toUIResourceObjects(tlr tiltfile.TiltfileLoadResult) typedObjectSet {
+	result := typedObjectSet{}
+
+	for _, m := range tlr.Manifests {
+		name := string(m.Name)
+		labels := m.Labels
+		labels[LabelOwnerKind] = LabelOwnerKindTiltfile // TODO(lizz): move this label assignment to the Tiltfile
```

Comment:
CI is complaining that you're trying to assign to a nil map here. what you probably need to do is make sure that all Manifests have non-nil label maps when you create them.